opencensus/stats/exporters/stackdriver_exporter.py

```diff
@@ -421,10 +421,10 @@ def get_task_value():
     """ getTaskValue returns a task label value in the format of
      "py-<pid>@<hostname>".
     """
-    task_value = "py@" + str(os.getpid())
     hostname = platform.uname()[1]
-    task_value += hostname if hostname is not None else "localhost"
-    return task_value
+    if not hostname:
+        hostname = "localhost"
```

Comment:
I'm surprised we default to localhost, but I know you didn't start the fire here.